version.h

```diff
@@ -22,7 +22,7 @@
 #define BOINC_VERSION_STRING "7.13.0"
 
 /* Package is a pre-release (Alpha/Beta) package */
-//#define BOINC_PRERELEASE 1
+#define BOINC_PRERELEASE 1
```

Comment:
FYI, that's fixed in 4aac6964243bbec095f4fafe313f4cd29ded5b71, further down in this PR.